packages/vuetify/src/components/VInput/VInput.tsx

```diff
@@ -42,7 +42,7 @@ export const makeVInputProps = propsFactory({
   appendIcon: IconValue,
   centerAffix: {
     type: Boolean,
-    default: true,
+    default: false,
```

Comment:
If default is false, can just do `centerAffix: Boolean` which defaults to false.